command_line_intro/files.md

```diff
@@ -0,0 +1,23 @@
+# Files and Directories
+
+The shell is a programming language that is run by the terminal. Like other programming languages, the shell:
+
+* Can collect many operations into single entities
+* Requires input
+* Produces output
+* Uses special characters
+* Has variables and state
+* Uses irritating syntax
+
+
+Additionally, as with programming languages, there are more shells than you’ll really care to learn. Among shells, bash is most widely used, so that is what we’ll use in this discussion. The [csh](), [tcsh](), [fish]()and [ksh]() shell types are also popular. Features of various shells are featured in the table below:
```

Comment:
Do we want to say which shells are commonly-used? I only see `sh` and `bash` installed on `ruby-office1`, for example.